addons/project/models/project.py

```diff
@@ -568,6 +571,33 @@ def _read_group_stage_ids(self, stages, domain, order):
     # customer portal: include comment and incoming emails in communication history
     website_message_ids = fields.One2many(domain=lambda self: [('model', '=', self._name), ('message_type', 'in', ['email', 'comment'])])
 
+    # recurring task fields
+    allow_recurring_tasks = fields.Boolean(related="project_id.allow_recurring_tasks", readonly=True)
```

Comment:
```suggestion
    allow_recurring_tasks = fields.Boolean(related="project_id.allow_recurring_tasks")
```
They are readonly by default